build/testing/cli.go

```diff
@@ -118,6 +118,18 @@ Run 'flipt --help' for usage.`))); err != nil {
 		}
 	}
 
+	{
+		container := container.
+			Pipeline("flipt").
+			WithExec([]string{"rm", "/etc/flipt/config/default.yml"})
+		if _, err := assertExec(ctx, container, flipt(),
+			fails,
+			stdout(contains(`loading configuration	{"error": "loading configuration: open /etc/flipt/config/default.yml: no such file or directory", "config_path": "/etc/flipt/config/default.yml"}`)),
+		); err != nil {
+			return err
+		}
```

Comment:
It might be nice to nest this in the block I referenced above.
Sincee this is to do with config location.

The `Pipeline` bit just gives the output a nice grouping.
The test could look like the following in the section above:
```go
if _, err := assertExec(ctx, container.WithExec([]string{"rm", "/etc/flipt/config/default.yml"}),
	flipt(),
	fails,
	stdout(contains(`loading configuration	{"error": "loading configuration: open /etc/flipt/config/default.yml: no such file or directory", "config_path": "/etc/flipt/config/default.yml"}`)),
); err != nil {
	return err
}
```